NOTES.md

```diff
@@ -27,6 +27,8 @@ _Backend Response_
 _Notes_ (specifically about the Backend)
 ```
 
+---
```

Comment:
lets wait till we have the final changes with the fronted stuff, before approve (then you dont need a new branch). looks all good till now :)

Reply:
@luke-dowling do we need to update again?